src/main/java/lotto/step1/model/Expression.java

```diff
@@ -0,0 +1,44 @@
+package lotto.step1.model;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Expression {
+    private final String expression;
+    private static Pattern customDelimiter;
+    static {
+        customDelimiter = Pattern.compile(Constants.customDelimiterRegex);
+    }
```

Comment:
```suggestion
    private static final Pattern customDelimiter = Pattern.compile(Constants.customDelimiterRegex);
```

굳이 static {} 에서 초기화를 하도록 한번 더 나누신 이유가 있으신가요?
의도가 궁금해서 여쭈어봅니다.